buckaroo/Tasks.fs

```diff
@@ -22,7 +22,6 @@ let private getCachePath = async {
 let getContext = async {
   let! cachePath = getCachePath
   //let git = new GitLib()
-
   let downloadManager = new DownloadManager(cachePath)
   let git = new GitLib()
```

Comment:
We should switch this on an environment variable. See https://docs.microsoft.com/en-us/dotnet/api/system.environment.getenvironmentvariable?view=netframework-4.7.2